README.md

```diff
@@ -1 +1,62 @@
-# capstone-1
\ No newline at end of file
+
+# Capstone project 1
+> This website created using this [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 
+
+## [](https://github.com/c3c1l1a/c3c1l1a.github.io#built-with)Built With
+-   HTML
+-   CSS
+- JS
+
+## [](https://github.com/c3c1l1a/c3c1l1a.github.io#getting-started)Getting Started
+
+**This  project is build with beautifly [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) originally used for the creative commons forum in 2015 in Soule South Korea.  If you want to take it and make your own, you can set it up in your local environment by following the steps below**
+
+To get a local copy up and running follow these simple steps.
+
+### [](https://github.com/c3c1l1a/c3c1l1a.github.io#prerequisites)Prerequisites
+
+-   Any modern browser (e.g Google Chrome)
+-   Text editor (e.g [Visual Studio Code](https://code.visualstudio.com/))
+
+### [](https://github.com/c3c1l1a/c3c1l1a.github.io#setup)Setup
+
```

Comment:
- [ ] Kindly record a video presenting the features of the project.
- [ ] Also deploy the page with gh pages, and add the link to your README.md

Reply:
Addressed 🔥